service_mesh/service_mesh_install/preparing-ossm-installation.adoc

```diff
@@ -22,6 +22,38 @@ include::modules/ossm-supported-configurations.adoc[leveloffset=+1]
 
 include::modules/ossm-installation-activities.adoc[leveloffset=+1]
 
+== Creating a mirror registry for installation on a restricted network
+
+If you performed a disconnected installation with a xref:../../installing/installing_restricted_networks/installing-restricted-networks-preparations.adoc#installing-restricted-networks-preparations[mirror registry in OpenShift], 
+you can add metadata to automatically pull containers for {ProductName}.
+
+.Containers
+
+Along with the containers required for Elasticsearch, the following containers are required for {ProductName}. Use the steps in xref:../../installing/installing_restricted_networks/installing-restricted-networks-preparations.adoc#installation-mirror-repository_installing-restricted-networks-preparations[Mirroring the OpenShift Container Platform image repository] 
+to create images for each {ProductName} registry.
+
+* registry.redhat.io/openshift-service-mesh/3scale-istio-adapter-rhel8:1.0.0
+* registry.redhat.io/openshift-service-mesh/citadel-rhel8:1.0.2
+* registry.redhat.io/openshift-service-mesh/istio-cni-rhel8:1.0.2
+* registry.redhat.io/openshift-service-mesh/galley-rhel8:1.0.2
+* registry.redhat.io/openshift-service-mesh/grafana-rhel8:1.0.2
+* registry.redhat.io/openshift-service-mesh/mixer-rhel8:1.0.2
+* registry.redhat.io/openshift-service-mesh/istio-rhel8-operator:1.0.2
+* registry.redhat.io/openshift-service-mesh/pilot-rhel8:1.0.2
+* registry.redhat.io/openshift-service-mesh/prometheus-rhel8:1.0.2
+* registry.redhat.io/openshift-service-mesh/proxyv2-rhel8:1.0.2
+* registry.redhat.io/openshift-service-mesh/sidecar-injector-rhel8:1.0.2
+* registry.redhat.io/openshift-service-mesh/istio-must-gather-rhel7:1.0.2
+* registry.redhat.io/openshift-service-mesh/proxy-init-rhel7:1.0.2
```

Comment:
3.11 isn't supported. Do we need to list the proxy-init-rhel7:1.0.2 as something that should be mirrored?

Reply:
Pulled that item out.